book/chapters/chapter9/preprocessing.qmd

```diff
@@ -12,36 +12,29 @@ In this book, preprocessing refers to everything that happens with *data* before
 
 `r index('Data cleaning', aside = TRUE)`\index{exploratory data analysis|see{data cleaning}} is an important part of preprocessing that involves the removal of errors, noise, and redundancy in the data; we only consider data cleaning very briefly as it is usually performed outside of `mlr3` on the raw dataset.
 
-Another aspect of preprocessing is `r index('feature engineering', aside = TRUE)`, which covers all other transformations of data before it is fed to the machine learning model, including the creation of features from possibly unstructured data, such as written text, sequences or images.
-The goal of feature engineering is to prepare the data so that a model can be trained on it, and/or to further improve predictive performance.
-It is important to note that feature engineering helps mostly for simpler algorithms, while highly complex models usually gain less from it and require little data preparation to be trained.
-Common difficulties in data that can be solved with feature engineering include features with (high) skew distributions, high cardinality categorical features, missing observations, high dimensionality and imbalanced classes in classification tasks.
```

Comment:
@mb706 @berndbischl  you've removed all definition of feature engineering? Now this paragraph is hard to follow

Reply:
I think this part got lost when transferring to gdoc and back, I will add it again